web/pages/_app.tsx

```diff
@@ -22,6 +23,13 @@ const fetcher = async (input: RequestInfo, init: RequestInit) => {
 
 function Inner(props: { pageProps: any; Component: any }) {
   const { graphqlApiUrl } = useServerConfig();
+  const { params } = props.pageProps;
+
+  useEffect(() => {
+    // enable the tools when the database is selected
+    window.ipc.updateAppMenu(params?.databaseName);
```

Comment:
Should this only be called if `process.env.NEXT_PUBLIC_FOR_ELECTRON === "true"`?